### DISCUSSION TITLE
Executive Director -  make audiences aware of our organizations's upcoming arts events

### DISCUSSION THREAD

**Discussion by mariel-marshall** [pair-cg/general-contributions#23] [User Stories]
### Original User Story: 
“As a executive director I want to have my online events show up properly in Google so that audiences can find my events to come to, even at the last minute. My staff will need to be able to enter this without a lot of extra work. Users need to be able to find my events and pay/sign up quickly.”

# 1. Need:
I want to work with my team in a simple and  efficient manner to inform potential audience members about the online performance / experience  

# 2. Action:
_What is an action that can be taken towards fulfilling that need?_
* post event info on organization website 
* post on social media
* send info to listing websites     
* send event to media for coverage

# 3. Desired Outcome:
_What is the desired outcome?_

Target audiences are reached
* audience members found the posting on the organization’s website (e.g., by serendipity)
* Google recommended organization’s  event to the appropriate audiences

# 4. Perspective:
_From what perspective is this user story coming from?_
Executive Director of an arts organization

# 5. Stakeholder Group:
_Which stakeholder group most closely fits the perspective of this User Story?_

Arts Organization

# 6. Additional info:
_Here's more information._
I want to make sure that Google can process correctly the information I post about the event start time and duration so that it can recommend the event to potential audiences that search for shows on the day of the event.

**Comment by mariel-marshall** [comment]
Questions for Jai: 

What aspects of the experience are you looking to communicate to audiences? Can you be specific here.

**Comment by fjjulien** [comment]
Regarding the findability of live performances via Google, it's important to note that Schema does unfortunately not have a type for "performing arts event" (but it does have sub-types for TheatreEvent, for example). The concept of "performance" exists in RDF ontologies (see the mapping to FRBRoo and mo in [wd:Q35140](https://www.wikidata.org/wiki/Q35140)), but is not directly represented in Schema. This might in part explain why it can be so difficult to search (and find) live performances on Google. Once the PAIR-CG modelling is complete and we have reached consensus on a class to represent live performances, we should propose a schema extension.
